src/library/importer.py

```
@@ -17,14 +24,61 @@ def import_quests_to(*, destination_schema, quest_import_ids):
+    Returns:
+        ImportResult: The result of the import operation, including row-level status info.
+
+    Raises:
+        ValueError: If the quest with the given import_id does not exist in the source schema
+                    or cannot be found in the destination schema after import.
+    """
+    with library_schema_context():
+        try:
+            quest = Quest.objects.get(import_id=quest_import_id)
+        except Quest.DoesNotExist():
+            raise ValueError(f"Failed to retrueve imported quest with import_id {quest_import_id}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)